tests/base/converters/test_dataframe_to_mds.py

```diff
@@ -56,6 +58,23 @@ def dataframe(self):
         df = spark.createDataFrame(data=data, schema=schema).repartition(3)
         yield df
 
+    @pytest.fixture
+    def array_dataframe(self):
+        spark = SparkSession.builder.getOrCreate()  # pyright: ignore
+
+        data = [([1, 2, 3], [1.0, 2.0, 3.0], [1e10, 1e11, 1e12]),
+                ([4, 5, 6], [4.0, 5.0, 6.0], [2e10, 2e11, 2e12]),
+                ([7, 8, 9], [7.0, 8.0, 9.0], [3e10, 3e11, 3e12])]
+
+        schema = StructType([
+            StructField('id', ArrayType(ShortType()), True),
+            StructField('dept', ArrayType(FloatType()), True),
+            StructField('properties', ArrayType(DoubleType()), True)
+        ])
+
+        df = spark.createDataFrame(data=data, schema=schema).repartition(3)
```

Comment:
just to clarify, what does `repartition` do?

Reply:
It just adding a process of redistributing rdd, more of a robustness test, not a must-have.